modules/emr_node/config/install/field.field.entity_meta_relation.node_meta_relation.emr_node_revision.yml

```diff
@@ -19,10 +19,8 @@ default_value_callback: ''
 settings:
   handler: 'default:node'
   handler_settings:
-    target_bundles:
-      - entity_meta_example_ct
```

Comment:
Shouldn't this turn into `target_bundles: { }` instead of being removed entirely? The same for the other field.